--- astrometry-0.67/Makefile.orig	2016-12-28 09:29:35.324301164 -0800
+++ astrometry-0.67/Makefile	2016-12-28 13:04:52.069165566 -0800
@@ -40,6 +40,8 @@
 #          qfits-an/libqfits.a -- FITS files
 #            util/libanbase.a  -- basic stuff
 
+STRIP ?= /opt/toolchain/toolchain-arm-linux/bin/arm-linux-strip
+
 include $(COMMON)/makefile.common
 #include $(COMMON)/makefile.qfits
 #include $(COMMON)/makefile.cfitsio
@@ -119,6 +121,49 @@
 	-($(MAKE) -C blind install-extra || echo "\nErrors in the previous make command are not fatal -- we try to build and install some optional code.\n\n")
 	@echo
 
+#
+# Minimum needed install for solving field 
+#
+MIN_BINS := util/an-pnmtofits blind/astrometry-engine blind/image2xy blind/solve-field
+
+# in util directory
+PYTHON_EXECS = image2pnm.py removelines.py uniformize.py __init__.py
+PYTHON_INSTALL = $(PYTHON_EXECS) addpath.py filetype.py fits.py run_command.py shell.py
+PY_INSTALL_DIR := $(INSTALL_DIR)/lib/python/astrometry
+PY_UTIL_DIR=$(PY_INSTALL_DIR)/util
+
+#
+# dynamic library in blind directory
+#
+ASTRO_LIB = libastrometry.so
+
+RUN_PY := runPython
+
+install-minimal-solve:
+	mkdir -p '$(PY_UTIL_DIR)'
+	mkdir -p '$(BIN_INSTALL_DIR)'
+	mkdir -p '$(LIB_INSTALL_DIR)'
+	cp __init__.py '$(PY_BASE_INSTALL_DIR)'
+	@for x in $(MIN_BINS); do \
+	  cp $$x $(BIN_INSTALL_DIR); \
+	done
+	@for x in $(PYTHON_INSTALL); do \
+	  cp util/$$x $(PY_UTIL_DIR); \
+	done
+	python -m compileall $(PY_UTIL_DIR)/*.py
+	rm $(PY_UTIL_DIR)/*.py
+	chmod +x $(PY_UTIL_DIR)/*.pyc
+	cp $(PY_UTIL_DIR)/__init__.pyc $(PY_INSTALL_DIR)
+	cp $(RUN_PY) $(BIN_INSTALL_DIR)
+	chmod +x $(BIN_INSTALL_DIR)/$(RUN_PY)
+	@for x in $(PYTHON_EXECS); do \
+	  (cd $(BIN_INSTALL_DIR); ln -sf ../lib/python/astrometry/util/$${x%%.*}.pyc .; ln -sf runPython $${x%%.*}.py ); \
+	done
+	mkdir -p $(LIB_INSTALL_DIR)/python2.7/site-packages
+	(cd $(LIB_INSTALL_DIR)/python2.7/site-packages; ln -sf ../../python/astrometry .)
+	cp blind/$(ASTRO_LIB) $(LIB_INSTALL_DIR)/
+	-$(STRIP) $(BIN_INSTALL_DIR)/* $(LIB_INSTALL_DIR)/*
+
 install-core:
 	mkdir -p '$(DATA_INSTALL_DIR)'
 	mkdir -p '$(BIN_INSTALL_DIR)'
